index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <title>Mint.com Sign Up Page Clone</title>
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        <div class="signin flexbox">
+            <div class="signin-text">
+                <div class="signin-line">Already have an Intuit account?</div>
+                <a href="#" class="signin-link"> I forgot my User ID or Password</a>
+            </div>
+            <div class="btn-class">
+                    <button class="signin-button"><i class="fas fa-unlock-alt icon"></i> Sign In</button>
+            </div>
+        </div>
+        <header class="header">
+            <h1 class="title">intuit</h1>
+            <div class="sub-headers">
+                <h5 class="sub-header">mint</h5>
+                <h5 class="sub-header">quickbooks</h5>
+                <h5 class="sub-header">turbotax</h5>
+            </div>
+        </header>
+        <div class="form-section">
+        <main class="main-section">
+            <div class="header-container">
+                <span class="description">One Account. <br> Everything Intuit.</span>
+                <div class="sign-up">Create an account for all of Intuit's products including Mint. <a href="#" class="learn-more">Learn more</a></div>
+            </div>
+            <form >
+            <fieldset class="fieldset">
+            <label class="label" for="email">
+                Email address
+            </label>
+            <input type="email" name="email" class="input" id="email" required>
+
+            
+            <label class="label" for="phone">
+                Phone <span>(recommended)</span>
+            </label>
+            <div class="number-input">
+                <input type="number" id="phone" class="phone-input" name="Phone">
+                <div class="warning">Standard call, messaging or data rates may apply.</div>
+           </div>   
+            <label class="label" for="password">
+                Password
+            </label>
+            <input type="text" id="password" class="input" name="Password">
+                
+        <button class="create" name="Submit">
+            <span><i class="fas fa-unlock-alt icon"></i>Create Account</span>
+        </button>
+    </fieldset>
+    </form>
+    <footer class="footer">
+            By clicking Create Account, you agree to our
+            <a href="#" class="footer-link">Terms of Service</a> and have read and acknowledge our <a href="#" class="footer-link">Privacy Statement.</a>
+    </footer>
+
+    </main>
+    <section class="captcha">
```

Comment:
sections should have a h1-h6 tag,if there is no heading you can use div instead.